docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`2414` Recover old balance proof to settle with partner using it
```

Comment:
A bit more info: If partner uses our old balance proof on-chain, the raiden client will now recover it from the WAL and properly use it on-chain.